tests/GenFu.Tests/TestEntities/Person.cs

```diff
@@ -26,7 +26,7 @@ private string _middleName
 
 
         // emailaddress EmailAddress email_address
-        public string EmailAddress { get; set; }
+        public String EmailAddress { get; set; }
```

Comment:
Sorry, this is fixed. I did this when checking the Type (String/string) would still work. Forgot to set it back to string. This is fixed and push up to my repos.